release/build.sh

```diff
@@ -70,3 +70,16 @@ mv -v opensearch-dsl*.gem $GIT_ROOT/dist/
 
 echo 'List gems to be published: '
 ls -l $GIT_ROOT/dist
+
+
+#####################################################
```

Comment:
Would this section not be before the 'List gems to be published' so that it can also be printed?

Reply:
You're right. Thanks for catching that